adapters/libuv.h

```diff
@@ -130,6 +130,9 @@ static void redisLibuvSetTimeout(void *privdata, struct timeval tv) {
 static void redisLibuvCleanup(void *privdata) {
     redisLibuvEvents* p = (redisLibuvEvents*)privdata;
 
+    if (p == NULL)
```

Comment:
1. It's not a good practice, to add such checks, because they could affect performance.
2. You couldn't check redundant cleanups in client code, if it's needed.
3. It's better to make early return before casting on line 131.

Reply:
Hi Mark
1. This is a best practice to check variables
2. This is internal function calls from driver
3. May be

P.s. Марк, ты вообще офигел :)
Я сейчас вспомню Серафим и всякое разное :)

Reply:
As i see, it's better to add check for nullability [here](https://github.com/redis/hiredis/blob/869f3d0ef1513dd0258ad7190c9914df16dcc4a4/async_private.h#L52), because there are [exist](https://github.com/search?q=repo%3Aredis%2Fhiredis%20ev.cleanup&type=code) another adapters, and their cleanup functions will be also called with NULL `privdata`.